pages/concepts/facilitator-fee.mdx

```diff
@@ -0,0 +1,396 @@
+---
+layout: concepts
+title: "Facilitator Fee"
+description: Collect a portion of the transfer amount on payments facilitated between two users.
+category: Features
+concept:
+  icon: "facilitator-fee.svg"
+meta:
+  title: Facilitator Fee for ACH transfers | Dwolla API Documentation
+  description: Charge users a flat rate amount to be removed from a payment as a fee. The fee is sent to the creator of the application.
+---
+
+# Facilitator fee
+
+The facilitator fee is a feature allowing for a flat rate amount to be removed from a payment as a fee, and sent to the creator of the Dwolla application. The fee does not affect the original payment amount, and exists as a separate Transfer resource with a unique transfer ID.
+
+#### Items to note before charging fees:
+
+- Fees can only be applied to a transaction between two parties
+- Facilitator fees can be debited from either the sending or receiving user of the payment
+- The sum of fees cannot exceed 50% of the original transfer amount
+- A facilitator fee must be at least \$0.01
```

Comment:
Noticed this `\$0.01` rendering as is. There's other files (not in this PR) that have these backslashes rendering, so I'll create a clean-up ticket to convert them to inline-code -`$0.01`.

Reply:
https://dwolla.atlassian.net/browse/WEB-702